components/sidebar/sidebar_category_list/index.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+
+import {getOrderedChannelIds} from 'mattermost-redux/selectors/entities/channels';
+import {getSidebarPreferences} from 'mattermost-redux/selectors/entities/preferences';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import SidebarCategoryList from './sidebar_category_list';
+
+// TODO: temp typing until we fix redux
+function mapStateToProps(state: GlobalState & {views: any}) {
+    // TODO: temp
+    const sidebarPrefs = getSidebarPreferences(state);
```

Comment:
We can probably remove anything that relies on the sidebar preferences at this point since we won't be using it going forward. Groups will be separate, sorting will be alphabetical, unreads won't be at the top, and favourites will be.

Reply:
This is copy and pasted from the original sidebar, and will be removed once we have the new redux structure.